mycore-base/src/test/java/org/mycore/datamodel/metadata/MCRObjectIDTest.java

```diff
@@ -50,7 +50,7 @@ public class MCRObjectIDTest extends MCRStoreTestCase {
      */
     public static void resetObjectIDFormat() {
         try {
-            Field fNumberformat = MCRObjectID.class.getDeclaredField("NUMBER_FORMAT");
+            Field fNumberformat = MCRObjectID.class.getDeclaredField("numberFormat");
```

Comment:
see MCRObjectID

Reply:
Can't  be final because its changed in a test by reflection.